.github/workflows/generate-preview.yml

```diff
@@ -29,6 +29,30 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-node-
 
+      - name: Remove create-preview label
+        uses: actions/github-script@0.9.0
+        with:
+          github-token: ${{secrets.GITHUB_TOKEN}}
+          script: |
+            github.issues.removeLabels({
+              issue_number: github.event.pull_request.number,
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              name: ['create-preview']
+            })
+
+      - name: Add generating-preview label
```

Comment:
I'd probably skip this part, since even though it will no-op, it will show up in the build results like on this preview as an extra "cancelled" action. Not sure if that result might also override the result of this successful run